node/process.ts

```diff
@@ -66,7 +66,15 @@ const argv = ["", "", ...Deno.args];
 // Overwrites the 1st item with getter.
 Object.defineProperty(argv, "0", { get: Deno.execPath });
 // Overwrites the 2st item with getter.
-Object.defineProperty(argv, "1", { get: () => fromFileUrl(Deno.mainModule) });
+Object.defineProperty(argv, "1", {
+  get: () => {
+    if (Deno.mainModule.startsWith("file:")) {
+      return fromFileUrl(Deno.mainModule);
+    } else {
+      return Deno.cwd() + "$deno$node.js";
```

Comment:
minor point, but maybe better to `path.join(Deno.cwd(), "$deno$node.js")` as `Deno.cwd()` doesn't end with path separator